reporting/slack.go

```diff
@@ -35,77 +36,45 @@ func (s *SlackReporter) BuildSummaryReport(
 	numRepos int,
 	report FindingSummary,
 	reportTime time.Time,
+	teamSummaries TeamSummaries,
 ) slack.Message {
 	reportBlocks := []slack.Block{
 		slack.NewHeaderBlock(
 			slack.NewTextBlockObject(slack.PlainTextType, header, false, false),
 		),
-		slack.NewDividerBlock(),
 		slack.NewContextBlock("", slack.NewTextBlockObject(
-			slack.PlainTextType, reportTime.Format(time.RFC1123), false, false,
+			slack.PlainTextType, reportTime.Format(DATE_LAYOUT), false, false,
```

Comment:
[q] same as above, are we moving away from standard RFC formats to omit time info?